doc/tutorials/SIR.ipynb

```diff
@@ -706,14 +716,31 @@
     "e_new = MSE_list[-1]/(I_UK[-1]-I_UK[-2])\n",
     "print(\"The percentage error of the SIR model over the last day reported cases is %.1f%%\" % (100*e_new))"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It must be noted that in the last day studied, the error is extremely low owing to an exceptionally good agreement on the last point of the data. Hence, a better estimate of the error in the predictions is to take the average percentage error on the cross validation subset, which considers from day 3 onwards."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eps_avg = np.mean(MSE_list/I_UK[-7:])*100\n",
+    "print(\"The average percentage deviation on the number of infected is %.1f%%\" % eps_avg)"
+   ]
   }
  ],
  "metadata": {
   "file_extension": ".py",
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.7.4 64-bit ('open-sir': pipenv)",
    "language": "python",
-   "name": "python3"
+   "name": "python37464bitopensirpipenv9b02a01dd5534b84a95035f2b67f676e"
```

Comment:
I don't understand this change haha

Reply:
This looks like what can be f***ing up the docs. Thanks for spotting! Looks like vscode nonsense.

Reply:
@sasalatart many thanks for all your comments. Very constructive and will improve the quality of the code. Tomorrow I will crack on

Reply:
This was the issue!